docs/Other SDL Features/Uploading Files/index.md

```diff
@@ -153,9 +153,9 @@ file.overwrite = true
 !@
 
 @![android, javaSE, javaEE]
```

Comment:
Would it be possible to delete the java description and reuse iOS's description by adding the java tags instead?